### ISSUE TITLE
Application destroyed message in bin/test

### ISSUE THREAD

**Issue by sudz123** [sympy/sympy#15223]
While running the bin/test we get the message in `test_plot.py` and `test_plot_implicit.py` : 

```"ttk::ThemeChanged"
can't invoke "event" command: application has been destroyed
    while executing
"event generate $w <<ThemeChanged>>"
    (procedure "ttk::ThemeChanged" line 6)
    invoked from within```

Is this a redundant test?

**Comment by asmeurer** [comment]
Can you give more context about where in the test the message is displayed. If you run the tests with `./bin/test plot -v` where does it show up?

**Comment by sudz123** [comment]
In `sympy/plotting/tests/test_plot.py` pops up with test_matplotlib_1,2,3,5,6 and in `sympy/plotting/tests/test_plot_implicit.py[2] ` it comes up with test_matplotlib. However there is a `ok` which is followed to everyone of them, so I am not sure whether it would affect anything.

**Comment by asmeurer** [comment]
We generally like to avoid things being printed to the terminal during the tests, but I haven't seen this before. It's possible there is some issue with your matplotlib installation. Does plotting work in an `./bin/isympy` session?

**Comment by sudz123** [comment]
Yes, I tried a basic : `plot((x**2),(x,-5,5))` and it works fine. Any other tests I should try?
